gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -380,7 +393,21 @@ def create(self, context):
     def update(self, context):
         context._plugin_context = self._get_resource_owner_context(
             context._plugin_context)
-        self._update(context)
+        network_function_map = self._get_node_instance_network_function_map(
+            context.plugin_session,
+            context.current_node['id'],
+            context.instance['id'])
+
+        if not all([network_function_map, context.original_node.get('config'),
+                    context.current_node.get('config')]):
```

Comment:
in _update method, you are checking for context.original_node.
So, in this if condition there is a possibility that, context.original_node.get('config') can fail with None type object has no attribute get.